docs/docs/code/nodejs/README.md

```diff
@@ -361,60 +364,46 @@ Sometimes you want to end your workflow early, or otherwise stop or cancel the e
 - You only want to run your workflow for users in the United States. If you receive a request from outside the U.S., you don't want the rest of the code in your workflow to run.
 - You may use the `user_id` contained in the event to look up information in an external API. If you can't find data in the API tied to that user, you don't want to proceed.
 
-**In any code step, calling the `$.flow.exit()` function will end the execution of the workflow immediately.** No remaining code in that step, and no code or destination steps below, will run for the current event.
+**In any code step, calling `return $.flow.exit()` will end the execution of the workflow immediately.** No remaining code in that step, and no code or destination steps below, will run for the current event.
 
 ```javascript
-defineComponent({
+export default defineComponent({
   async run({ steps, $ }) {
-    $.flow.exit();
+    return $.flow.exit();
```

Comment:
I always forget that using `$.flow.exit()` requires an explicit `return`. Not a great DX but you can talk to Giao about this one... we've faced some challenges here.

Reply:
Oh wow I didn't know that. My other examples are broken then. I thought it was the same behavior as `$.end`